docs/computing/jobs/billing.md

```diff
@@ -1,44 +1,55 @@
-# Billing
+[lumi-c]: ../../computing/systems/lumic.md
 
-Running jobs and storing data on the parallel filesystem consume the billing 
-units allocated to your project:
+# Billing policy
 
-- for computing, your project is allocated CPU-core-hours 
-- for storage, your project is allocated GB-hours.
+Running jobs on the compute nodes and storing data on the storage will consume
+the billing units allocated to your project:
+
+- Compute is billed in units of CPU-core-hours for CPU-nodes and GPU-hours for GPU-nodes.
+- Storage is billed in units of TB-hours.
 
 ## Compute billing
 
-For compute, your project is allocated CPU-core-hours that are consumed when 
-running jobs. Depending on the partition, the way this billing is carried out 
+### CPU billing
+
+For compute, your project is allocated CPU-core-hours that are consumed when
+running jobs. Depending on the partition, the way this billing is carried out
```

Comment:
```suggestion
running jobs. Depending on the SLURM partition, the way this billing is carried out
```

Reply:
I have updated the entire page to explicitly use the wordring "Slurm partition"